README.md

```diff
@@ -114,12 +151,17 @@ To start or stop individual daemons, the sysvinit script can be used:
 Running unit tests
 ==================
 
-To run build and run all tests, use ctest:
+To run build and run all tests run in `make check`, use ctest:
```

Comment:
I'm guessing "run build" means simply "build" unless _build_ is a command. Would this be better?

To build and run all tests that are executed during `make check`, use `ctest`: